src/GovUk.Education.ExploreEducationStatistics.Common/Model/Chart/Charts.cs

```diff
@@ -59,6 +59,9 @@ public class VerticalBarChart : Chart
     public class MapChart : Chart
     {
         public override ChartType Type => Map;
+        
+        // TODO EES-3319 - make mandatory when all Map Charts are migrated to have a Boundary Level set
+        public long? BoundaryLevel { get; set; }
```

Comment:
I think it's worth building something into the test plan for this so that we don't forget, that a full content refresh is required after running the migration (i.e. the destructive one run by the Publisher that clears all Release json and then recreates it).

The cache clearing mechanism that we recently introduced for clearing cached data blocks from the public blob cache actually clears cached data block (table) results.

The data block config here that is being altered by the migration won't change because it's cached within Release blobs and they don't get cleared.

Reply:
Cool cheers, I have included that in the PR notes above.